pedpy/methods/profile_calculator.py

```diff
@@ -68,6 +69,35 @@ class SpeedMethod(Enum):  # pylint: disable=too-few-public-methods
     is the number of pedestrians inside :math:`P_M` (:math:`|P_M|`).
     """
 
+    GAUSSIAN = auto()
+    r"""Compute Gaussian speed profile.
+    
+    In each cell the weighted speed :math:`v_{c}` is calculated as 
+    
+    .. math::
+     
+        v_{c} = \frac{\sum_{i=1}^{N}{\big(w_i(\delta)\cdot v_i\big)}}{\sum_{i=1}^{N} w_i},
+
+    where :math:`v_i` is the speed of a pedestrian and :math:`w_i` are weights 
+    depending on the pedestrian's distance :math:`\delta` from its position 
+    (:math:`\boldsymbol{r}_i`) to the center of the grid 
+    (:math:`\boldsymbol{c}`) cell:
+
+    .. math::
+        \delta =  \boldsymbol{r}_i - \boldsymbol{c}.
+    
+    The weights :math:`w_i` are  calculated by a Gaussian as follows:
+   
+    .. math::
+    
+        w_i = \frac{1} {\sigma \cdot \sqrt{2\pi}} \exp\big(-\frac{\delta^2}{2\sigma^2}\big),
```

Comment:
Isn't this $w_i(\delta)$?

Reply:
Yes, I think I'll keep $w_i$ everywhere for simplicity.